app/src/features/settings/components/BillingTeam/components/BillingDetails/components/ReviewJoinRequestModal/ReviewJoinRequestModal.scss

```diff
@@ -0,0 +1,67 @@
+.review-request-modal {
+  & .ant-modal-header {
```

Comment:
No need of `&` since `.ant-modal-header` is already nested inside `.review-request-modal` class.